W-14901181 fins 1.10 beta to latest (#79)
mulesoft/docs-accelerators: financial-services/1.10/modules/ROOT/pages/upgrade-notes.adoc

=== Change Summary

The following changes may have an impact on existing deployments:

* The CIM libraries were upgraded to new versions containing better examples and simplified (but more complete) JSON schemas.
* The Notifications Process API is now referenced as a shared service.
* Support for additional Salesforce FSC record types has been added.
* The ID Graph LWC has been updated.

=== Upgrade Steps

If a release earlier than 1.3, incorporate the previous upgrade steps into the following ones.

. Apply the latest configuration changes to your Salesforce Financial Services Cloud instance as documented in the customization guide.
. Deploy the latest ID Graph LWC asset to your target Salesforce org(s).
. Apply the latest schema changes to the Core Banking and MDM databases.
** This may require some updates to existing records.
. Deploy the latest customer sync components included in this release.

=== Verification Steps

> * Create a new customer in Salesforce FSC, ensure it is replicated to Core Banking and MDM.

Reviewer: This isn't a complete sentence as written. Either "To create ..." or "FSC; ensure"

> * Update the same customer in FSC, ensure the update is reflected in Core Banking and MDM.

Reviewer: Same as the above

* Ensure the Customer ID Graph component displays all target systems correctly.
* Repeat these tests for financial accounts, transactions, and cards in FSC.
* Repeat the above steps for customers, accounts, transactions, and cards in Core Banking.
* Walk through the Loan Origination scenario and ensure customer and loan account are properly created in FSC, Core Banking, and MDM.

== Release 1.2 to Release 1.3

In addition to new components introduced for the Payments Modernization use case, this release also includes updates to existing Core Banking, MDM, and Salesforce integration components.

=== Change Summary

The following changes may have an impact on existing deployments:

> * Objects in the Banking Library and associated database schemas have been expanded.

Reviewer: ```suggestion
* Objects in the Banking Library and associated database schemas are expanded.
```

> * Additional support for Organization parties in the customer sync process was introduced.

Reviewer: ```suggestion
* Additional support for Organization parties in the customer sync process is introduced.
```
